examples/pipeline/dummy_entity_linking.py

```diff
@@ -0,0 +1,71 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+"""Demonstrate how to build a simple knowledge base and run an Entity Linking algorithm.
+Currently still a bit of a dummy algorithm: taking simply the entity with highest probability for a given alias
+"""
+import spacy
+from spacy.kb import KnowledgeBase
+
+
+def create_kb(vocab):
+    kb = KnowledgeBase(vocab=vocab)
+
+    # adding entities
+    entity_0 = "Q1004791_Douglas"
+    print("adding entity", entity_0)
+    kb.add_entity(entity=entity_0, prob=0.5)
+
+    entity_1 = "Q42_Douglas_Adams"
+    print("adding entity", entity_1)
+    kb.add_entity(entity=entity_1, prob=0.5)
+
+    entity_2 = "Q5301561_Douglas_Haig"
+    print("adding entity", entity_2)
+    kb.add_entity(entity=entity_2, prob=0.5)
+
+    # adding aliases
+    print()
+    alias_0 = "Douglas"
+    print("adding alias", alias_0)
+    kb.add_alias(alias=alias_0, entities=[entity_0, entity_1, entity_2], probabilities=[0.1, 0.6, 0.2])
```

Comment:
I find the name `alias` a bit confusing. I would call this `mention` or `mention_text` or even `candidate_mention_text`, and I would use `aliases` to describe the list of alternative names for the entity. While they might be mappable in certain situations, they come from different sources and can serve different purposes.

Reply:
So the `alias` here is in fact an alternative name for a set of given entities. The reason why we initially opted not to add the aliases when defining an entity (cf. your other comment), is because we want to facilitate defining prior probabilities `P(entity|alias)`. By adding these alias-by-alias, we can enforce that the sum of these prior probabilities (over all possible entities for 1 alias/mention) does not exceed 1, which is more difficult when you add the aliases at the same time when defining one entity. Does that make sense?

Reply:
I agree with you though that if you don't have prior probabilities, adding aliases when defining the entities will be more intuitive...

Reply:
The problem with `mention` to me is it sounds like a *token*, when we want a *type* here. I think we need to keep the term `mention` for an instance of an entity in text. An `alias` would be the text that anchors a mention. I agree that it might not be what you first think of, but at least it will let us keep consistent terminology.

> I agree with you though that if you don't have prior probabilities, adding aliases when defining the entities will be more intuitive...

Yeah, but we want to make sure there's only one way to do it. So I think this API is better overall.